nservicebus/upgrades/7to8/index.md

```diff
@@ -121,11 +121,19 @@ NServiceBus no longer natively supports forwarding a copy of every message proce
 
 The `NServiceBus.Host` package is deprecated. See the [NServiceBus Host upgrade guide](/nservicebus/upgrades/host-7to8.md) for details and alternatives.
 
-
 ## NServiceBus Azure Host
 
 The `NServiceBus.Hosting.Azure` and `NServiceBus.Hosting.Azure.HostProcess` are deprecated.See the [NServiceBus Azure Host upgrade guide](/nservicebus/upgrades/acs-host-7to8.md) for details and alternatives.
 
+## Overriding the host machine name
+
+The `RuntimeEnvironment.MachineNameAction` method has been deprecated. To override the machine name of the endpoint instance use:
```

Comment:
`RuntimeEnvironment.MachineNameAction` is a property, not a method. But I think we can simplify anyway:

```suggestion
`RuntimeEnvironment.MachineNameAction` is deprecated. The host machine name of an endpoint instance may be overridden using:
```

Reply:
LGTM